libpod/container_internal.go

```diff
@@ -1410,14 +1410,16 @@ func (c *Container) cleanupStorage() error {
 	var cleanupErr error
 
 	for _, containerMount := range c.config.Mounts {
-		if err := c.unmountSHM(containerMount); err != nil {
-			if cleanupErr != nil {
-				logrus.Errorf("Error unmounting container %s: %v", c.ID(), cleanupErr)
-			}
-			cleanupErr = err
+		if err := c.unmountExtra(containerMount); err != nil {
+			logrus.Errorf(err.Error())
 		}
 	}
 
+	if err := c.cleanupOverlayMounts(); err != nil {
```

Comment:
This is already called by unmountExtra; we should either remove the call to it in unmountExtra or remove the call here

Reply:
No it it is not?  I don't see it in there.  There is a 
	if err := overlay.CleanupMount(mount); err != nil {

Which is different then.
	return overlay.CleanupContent(c.config.StaticDir)